### ISSUE TITLE
Using edsl for code generation  - error (and add documentation)

### ISSUE THREAD

**Issue by johnjosephhorton** [expectedparrot/edsl#425]
<img width="1250" alt="Screenshot 2024-05-04 at 10 12 04 AM" src="https://github.com/expectedparrot/edsl/assets/407218/df771e66-3e1f-4cab-b4dd-7a37359832ba">

**Comment by rbyh** [comment]
@zer0dss I think you worked on a similar issue where """ """ tripe quotes created a problem?

**Comment by zer0dss** [comment]
@rbyh Yes, I think we had a similar issue on xdsl. I have to check it.
